openapi-maven-plugin/src/test/resources/ut/SpringClassAnalyserTest/generic_parent_bound_by_child.yml

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
I think that this file, and the other one `parametrized_object_class.yml,`are not used